gradle-conjure-api/src/main/java/com/palantir/gradle/conjure/api/ConjureExtension.java

```diff
@@ -27,26 +27,31 @@ public class ConjureExtension {
 
     private final GeneratorOptions typescriptOptions = new GeneratorOptions();
     private final GeneratorOptions javaOptions = new GeneratorOptions();
+    private final GeneratorOptions parserOptions = new GeneratorOptions();
     private final GeneratorOptions pythonOptions = new GeneratorOptions();
     private final Map<String, GeneratorOptions> genericOptions = new HashMap<>();
 
-    public final void typescript(@DelegatesTo(GeneratorOptions.class) Closure<GeneratorOptions> closure) {
+    public final <T> void typescript(@DelegatesTo(GeneratorOptions.class) Closure<T> closure) {
```

Comment:
Hmm, I wonder if parameterising the value like this works in the gradle integration or it really does have to be a raw type like I've been doing.

Reply:
Sorry, I misinterpreted  your comment, my fault -- fixed